sovtoken/sovtoken/test/helpers/helper_general.py

```diff
@@ -53,9 +53,11 @@ def do_nym(
         request = self._request.nym(seed, alias, role, dest, verkey, sdk_wallet)
         return self._send_get_first_result(request)
 
-    def do_transfer(self, inputs, outputs, identifier=None):
+    def do_transfer(self, inputs, outputs, identifier=None, extra=None, text=None, mechanism=None, version=None):
         """ Build and send a transfer request. """
-        request = self._request.transfer(inputs, outputs, identifier=identifier)
+        if text and mechanism and version:
+            extra = self._request.add_transaction_author_agreement_to_extra(extra, text, mechanism, version)
```

Comment:
Is it ok that we may pass `None` extra into `add_transaction_author_agreement_to_extra`?

Reply:
It is. We might want to prepare extra with only TAA Aceptance things